accounts/abi/bind/precompiletemplate.go

```diff
@@ -0,0 +1,366 @@
+// (c) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package bind
+
+// tmplPrecompileData is the data structure required to fill the binding template.
+type tmplPrecompileData struct {
+	Contract *tmplPrecompileContract // The contract to generate into this file
+	Structs  map[string]*tmplStruct  // Contract struct type definitions
+}
+
+// tmplPrecompileContract contains the data needed to generate an individual contract binding.
+type tmplPrecompileContract struct {
+	*tmplContract
+	AllowList bool                   // Indicator whether the contract uses AllowList precompile
+	Funcs     map[string]*tmplMethod // Contract functions that include both Calls + Transacts in tmplContract
+}
+
+// tmplSourcePrecompileGo is the Go precompiled source template.
+const tmplSourcePrecompileGo = `
+// Code generated
+// This file is a generated precompile contract with stubbed abstract functions.
+// The file is generated by a template. Please inspect every code and comment in this file before use.
+
+// There are some must-be-done changes waiting in the file. These are highlighted by CAPITAL WORDS.
```

Comment:
```suggestion
// There are some must-be-done changes waiting in the file. Each area requiring you to add your code is marked with CUSTOM CODE to make them easy to find and modify.
```

Reply:
Want to make it easy for users to search for them instead of needing to scan the file looking for capital words